### PR TITLE
Support IAM caps

### PR DESCRIPTION
**Pull request by dejan2k**
What              | Where/Who
------------------|----------------------------------------
JIRA Issue        | [SDK-412](https://leanplum.atlassian.net/browse/SDK-412)
People Involved   | @dejan2k

### CODE REVIEW

File: Leanplum-SDK/Classes/Features/Actions/LPActionManager.m [hunk 1]
```diff
@@ -498,4 +533,112 @@ - (void)muteFutureMessagesOfKind:(NSString *)messageId
...
+{
+    int dayLimit = 0;
+    int weekLimit = 0;
+    int sessionLimit = 0;
+    
+    for (NSDictionary *cap in [[LPVarCache sharedCache] getLocalCaps]) {
+       if (![@"IN_APP" isEqualToString:[cap valueForKey:@"channel"]]) {
+         continue;
+       }
+        NSString *type = (NSString *)[cap valueForKey:@"type"];
```
Comment (milos1290) [inline Leanplum-SDK/Classes/Features/Actions/LPActionManager.m:550]: indentation and formatting

**Review by milos1290 — CHANGES_REQUESTED** [review]

File: Leanplum-SDK/Classes/Features/Actions/LPActionManager.m [hunk 2]
```diff
@@ -498,4 +533,112 @@ - (void)muteFutureMessagesOfKind:(NSString *)messageId
...
+*/
+- (BOOL)shouldSuppressMessages
+{
+    int dayLimit = 0;
+    int weekLimit = 0;
+    int sessionLimit = 0;
+    
+    for (NSDictionary *cap in [[LPVarCache sharedCache] getLocalCaps]) {
+        if (![@"IN_APP" isEqualToString:[cap valueForKey:@"channel"]]) {
+            continue;
```
Comment (nzagorchev) [inline Leanplum-SDK/Classes/Features/Actions/LPActionManager.m:548]: Add logging, for example:
                    LPLog(LPDebug, @"Local IAM caps reached, suppressing messageId=%@", [actionContext messageId]);
                    continue;

File: Leanplum-SDK/Classes/Internal/Leanplum.m [hunk 3]
```diff
@@ -1899,10 +1902,17 @@ + (void)maybePerformActions:(NSArray *)whenConditions
                  recordHeldBackImpression:[actionContext messageId]
                  originalMessageId:[actionContext originalMessageId]];
             } else {
+                if ([self shouldSuppressMessage:actionContext]) {
+                    continue;
```
Comment (nzagorchev) [inline Leanplum-SDK/Classes/Internal/Leanplum.m:1906]: Add logging, for example:
```
LPLog(LPDebug, @"Local IAM caps reached, suppressing messageId=%@", [actionContext messageId]);
continue;
```

**Review by nzagorchev — APPROVED** [review]
Looks good. E2E tested.

**Pull request merged**